public/static/app/perfdiscovery/PerfDiscoveryDataService.js

```diff
@@ -121,6 +122,12 @@ mciModule.factory('PerfDiscoveryDataService', function(
     }
   }
 
+  // :param key: String (e.g. 'BF-123')
+  // :returns: String (e.g. 'https://jirahost/browse/BF-123')
+  function jiraLink(key) {
```

Comment:
I would instead create a new filter function that forms this link and change the existing jiraLinkify function to use the new function so that we don't duplicate this